src/components/Observations/map/TooltipContents.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+
+type TooltipContentsProps = {
+  title: string;
+  observationInProgress: boolean;
+  numPlants?: number;
+  numSpecies?: number;
+  plantingDensity?: number;
+  percentMortality?: number;
+};
+
+export default function TooltipContents(props: TooltipContentsProps): JSX.Element {
+  const { title, observationInProgress, numPlants, numSpecies, plantingDensity, percentMortality } = props;
+  const theme = useTheme();
+
+  return (
+    <Box display='flex' flexDirection='column' padding={theme.spacing(1)}>
+      <Typography fontSize='16px' fontWeight={600} marginBottom={theme.spacing(2)}>
+        {title}
+      </Typography>
+      {observationInProgress ? (
+        <Typography>{strings.OBSERVATION_IN_PROGRESS}</Typography>
+      ) : (
+        <>
+          <Typography fontSize='16px' fontWeight={400}>
+            {`${numPlants ?? strings.UNKNOWN} ${strings.PLANTS}`}
+          </Typography>
+          <Typography fontSize='16px' fontWeight={400}>
+            {`${numSpecies ?? strings.UNKNOWN} ${strings.SPECIES}`}
+          </Typography>
+          <Typography fontSize='16px' fontWeight={400}>
+            {`${strings.PLANTING_DENSITY}: ${plantingDensity ?? strings.UNKNOWN} ${strings.PLANTS_PER_HECTARE}`}
+          </Typography>
+          <Typography fontSize='16px' fontWeight={400}>
+            {`${strings.MORTALITY_RATE}: ${percentMortality ?? strings.UNKNOWN}%`}
+          </Typography>
```

Comment:
should we have the quantities on the left hand side? I understand the suffixes plants/ha and % make this confusing. Or maybe have the quantities for the others show up on the right hand side, just to be consistent. Agree, this is a design decision.

Reply:
I think you're right. Maybe it should be "Label: quantity" for all items

Reply:
That might look cleaner with suffixes/units. Something to iterate on with design input perhaps.

Reply:
I'll try that and update the screenshots 👍